packages/terra-framework-docs/src/terra-dev-site/doc/popup/example/PopupWithArrow.jsx

```diff
@@ -18,25 +23,65 @@ function PopupArrow() {
 
   const handleRequestClose = () => {
     setOpen(false);
+    buttonElement.current.focus(); // Return focus to the button when the popup closes
+  };
+
+  const handleKeyDown = (event) => {
+    if (event.key === 'ArrowDown') {
+      event.preventDefault();
+      setFocusedIndex((prevIndex) => (prevIndex + 1) % 2);
+    } else if (event.key === 'ArrowUp') {
+      event.preventDefault();
+      setFocusedIndex((prevIndex) => (prevIndex - 1 + 2) % 2);
+    }
   };
+  useEffect(() => {
+    if (open) {
+      setFocusedIndex(0);
+    }
+  }, [open]);
+
+
+  useEffect(() => {
+    if (open && focusedIndex !== -1) {
+      const activeMenuItem = document.getElementById('activeMenuItem');
+      if (activeMenuItem) {
+        activeMenuItem.focus();
+      }
+    }
+  }, [open, focusedIndex]);
 
   return (
     <React.Fragment>
-      <Button text="Medicaton Links" onClick={handleButtonClick} refCallback={setButtonNode} />
+      <Button text="Medication Links" onClick={handleButtonClick} refCallback={setButtonNode} />
       <Popup
         isOpen={open}
         onRequestClose={handleRequestClose}
         targetRef={getButtonNode}
         isArrowDisplayed
       >
-
-        <a href="https://www.refseek.com/directory/health_medical.html">Health and Medical sites</a>
-        <br />
-        <a href="https://www.northeaststate.edu/Campus-Life/Health-and-Wellness/Health-Related-Links/
-        "
-        >
-          Health Related Links
-        </a>
+        <ul className={cx('terra-popup-content')} onKeyDown={handleKeyDown} role="menu">
+          <li tabIndex={0} role="menuitem">
```

Comment:
I think we should remove tabIndex from li to avoid double focus on same item. (it first focuses `li` and then `link` )

Reply:
Resolved @supreethmr  "https://github.com/cerner/terra-framework/pull/1660/commits/ff6fdf4bec6b90ceae1bc894cffb9aaebe5ff2ed"